e2e/cypress/support/api/role.js

```diff
@@ -198,12 +216,30 @@ export const defaultRolesPermissions = {
         'sysconsole_write_user_management_groups',
         'read_private_channel_groups',
         'delete_public_channel',
-        'sysconsole_write_site',
+        'sysconsole_write_site_customization',
+        'sysconsole_write_site_localization',
+        'sysconsole_write_site_users_and_teams',
+        'sysconsole_write_site_notifications',
+        'sysconsole_write_site_announcement_banner',
+        'sysconsole_write_site_emoji',
+        'sysconsole_write_site_posts',
+        'sysconsole_write_site_file_sharing_and_downloads',
+        'sysconsole_write_site_public_links',
+        'sysconsole_write_site_notices',
         'sysconsole_read_user_management_channels',
         'sysconsole_read_user_management_teams',
         'sysconsole_write_user_management_permissions',
         'sysconsole_read_about',
-        'sysconsole_read_site',
+        'sysconsole_read_site_customization',
+        'sysconsole_read_site_localization',
+        'sysconsole_read_site_users_and_teams',
+        'sysconsole_read_site_notifications',
+        'sysconsole_read_site_announcement_banner',
+        'sysconsole_read_site_emoji',
+        'sysconsole_read_site_posts',
+        'sysconsole_read_site_file_sharing_and_downloads',
+        'sysconsole_read_site_public_links',
+        'sysconsole_read_site_notices',
```

Comment:
Hey Anurag,

Please don't forget to give any ancillary permissions that are associated to these! Thanks.